isis/src/qisis/apps/qtie/QtieTool.cpp

```diff
@@ -916,18 +916,18 @@ namespace Isis {
 
     // Update the cube history
     p_matchCube->write(*cmatrix);
-    History h("IsisCube");
-    try {
-      p_matchCube->read(h);
-    }
-    catch (IException &e) {
-      QString message = "Could not read cube history, "
-                        "will not update history.\n";
-      QString errors = e.toString();
-      message += errors;
-      QMessageBox::warning((QWidget *)parent(), "Warning", message);
-      return;
-    }
+    History h = p_matchCube->readHistory("IsisCube");
+    // try {
```

Comment:
Is this block something that can be removed?

Reply:
Probably? I commented this out as I didn't know how to add the functionality back in. I think checking the potential read in history for a number of objects > 0 would probably work?

Reply:
I think this can go